apps/portal/controllers/apis/login.jag

```diff
@@ -0,0 +1,43 @@
+<%
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+(function () {
+        var log = new Log();
+        var username = request.getParameter("username");
+        var password = request.getParameter("password");
+        var usr = require('/modules/user.js');
+        var api = require('/modules/api.js');
+        var carbon = require('carbon');
```

Comment:
better be consistent with quotations for defining strings, use double quotes for outer strings and single quotes for inner strings.